src/XamlStyler.Console/XamlStylerConsole.cs

```diff
@@ -12,6 +12,7 @@ namespace Xavalon.XamlStyler.Console
 {
     public sealed class XamlStylerConsole
     {
+        private static readonly string[] SupportedPatterns = { "*.xaml", "*.axaml" };
```

Comment:
The asterisk wildcard is required for retrieving all matching XAML and AXAML files in a specified directory (line 192) but cannot be there when checking the file extension when processing of the file itself (line 241).

Reply:
Sorry, thought I had tested that